src/xcode/ENA/ENA/Resources/Environment/Environments.default.json

```diff
@@ -7,6 +7,8 @@
 			"verificationURL": "https://verification.coronawarn.app",
 			"dataDonationURL": "https://data.coronawarn.app",
 			"errorLogSubmissionURL": "https://logupload.coronawarn.app",
+			"healthCertificateProofURL": "https://api.recertify.ubirch.com",
+			"healthCertificatePinningKeyHash": "d642f00397b8eca34571d12f67913192059c4fc01eab9acc0030aa3477b4ee15",
```

Comment:
Have you updated the environments repo as well? https://github.com/corona-warn-app/cwa-app-ios-environments
(this is the blocker to prevent issues on Testflight build)

Reply:
https://github.com/corona-warn-app/cwa-app-ios-environments/pull/1